dp-hello-world-controller/ci/scripts/audit.sh

```diff
@@ -1,15 +1,7 @@
----
-platform: linux
+#!/bin/bash -eux
 
-image_resource:
-  type: docker-image
-  source:
-    repository: onsdigital/dp-concourse-tools-nancy
-    tag: latest
+export cwd=$(pwd)
```

Comment:
We actually don't need to export a variable for current location on disk; we can remove this and it's use on line 5:

```suggestion
```